src/app/organizations/state/register-organization.service.ts

```diff
@@ -178,6 +189,7 @@ export class RegisterOrganizationService {
         location: organizationFormState.location,
         email: organizationFormState.contactEmail,
         status: Organization.StatusEnum.PENDING,
+        avatarUrl: organizationFormState.avatarUrl || null,
```

Comment:
The `|| null` is redundant, unless you're trying to catch an undefined case and/or you want an empty string to become null.

Reply:
I do want the empty string to become null. If it's an empty string, the database will reject it because of the regex. So I can either have null here, or change the regex in the the webservice to accept an empty string.

Reply:
:+1: